src/dataLoaders/components/collectorsWizard/select/StreamingSelector2.tsx

```diff
@@ -188,6 +189,7 @@ class StreamingSelectorTelegrafUiRefresh extends PureComponent<Props, State> {
       configured: ConfigurationState.Configured,
     }
     this.props.onTogglePluginBundle(pluginBuild)
+    event(`telegraf_page.create_new_config.${plugin}_plugin_selected`)
```

Comment:
we can alternatively do something like:

```ts
event('telegraf_page.create_new_config', {plugin})
```

and the plugin should show up as a tag.